packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+            color: #A2A2A8;
+        }
+
+        .v-center {
+            position: absolute;
+            top: 50%;
+            transform: translate(0, -50%);
+        }
+
+        .icon {
+            margin-bottom: 20px  ;
+        }
+
+        form {
+            width: 100%;
+        }
+
+        .full-width {
+            width: 100%;
+        }
+
+
+        .modal-footer {
+            display: flex;
+            align-items: right;
+            justify-content: flex-end;
+            padding-top: 0px;
+            padding-bottom: 0px;
+
+            button {
+                width: 136px  ;
+                height: 56px  ;
+                margin-top: 0px  ;
+
+                &.link {
+                    margin: 20px 35px;
+                }
+                
+                &.blue {
+                    padding: 0 35px;
+                }
+            }
+        }
+
+        .success-bottons {
+            align-items: center;
+            margin-left: auto;
+            margin-right: auto;
+            margin-top: 25px;
+            margin-bottom: 25px;
+        }
+    }
+`;
+
+export default function NFTTransferModal({ open, onClose, nft, accountId }) {
+    const [ receiverId, setReceiverId ] = useState('');
+    const [ result, setResult ] = useState();
+    const [ sending, setSending ] = useState(false);
+    const [ viewType, setViewType ] = useState('transfer');
+    const [ accountIdIsValid, setAccountIdIsValid] = useState(false);
+    const balance = useSelector(state => selectBalance(state));
+    const nearBalance = balance.balanceAvailable;
+    const dispatch = useDispatch();
+    const { updateNFTs } = nftActions;
+
+    const { localAlert } = useSelector(({ status }) => status);
+
+    function onTransferSuccess(result, newOwnerId) {
+        setResult(result);
+        dispatch(updateNFTs({ accountId }));
+        setViewType('success');
+    }
+
+    async function sendNFT (nft, receiverId) {
+        setSending(true);
+        try {
+            const { contract_id, token_id, owner_id } = nft;
+            const res = await NonFungibleTokens.transfer({
+                accountId: owner_id,
+                contractId: contract_id,
+                tokenId: token_id,
+                receiverId
+            });
+
+            onTransferSuccess(res, receiverId);
+        } catch (err) {
+            dispatch(showCustomAlert({
+                success: false,
+                messageCodeHeader: 'error',
+                messageCode: 'walletErrorCodes.sendNonFungibleToken.error',
+                errorMessage: err.message,
+            }));
+        } finally {
+            setSending(false);
+        }
+    }
+
+    return (
+        <Modal
+            id='nft-transfer-modal'
+            isOpen={nft}
+            onClose={onClose}
+            closeButton={false}
+            modalSize='md'
+        >
+            {viewType === 'transfer' &&
+            <StyledContainer className='small-centered'>
```

Comment:
```suggestion
            <StyledContainer>
```
`small-centered` isn't doing anything here